pkg/prometheus/promcfg_test.go

```diff
@@ -5333,6 +5333,20 @@ func TestScrapeConfigSpecConfig(t *testing.T) {
 			},
 			golden: "ScrapeConfigSpecConfig_DNSSD_ARecord.golden",
 		},
+		{
+			name: "enable_compression_is_set_to_true",
+			scSpec: monitoringv1alpha1.ScrapeConfigSpec{
+				EnableCompression: ptr.To(true),
+			},
+			golden: "ScrapeConfigSpecConfig_EnableCompression.golden",
+		},
+		{
+			name: "enable_compression_is_set_to_false",
+			scSpec: monitoringv1alpha1.ScrapeConfigSpec{
+				EnableCompression: ptr.To(false),
+			},
+			golden: "ScrapeConfigSpecConfig_EnableCompression2.golden",
```

Comment:
It would be good to name golden file as the testcase so it can even function as a example to check 

```suggestion
			golden: "ScrapeConfigSpecConfig_EnableCompression_False.golden",
```

Reply:
Thanks. Fixed it.